docs/approvals/service-now/index.md

```diff
@@ -5,7 +5,7 @@ position: 10
 ---
 
 :::warning
-The ServiceNow Integration feature is available from Octopus **2022.3** onwards and requires an [enterprise license](https://octopus.com/enterprise).
+The ServiceNow Integration feature is available from Octopus **2022.3** onwards and requires an [enterprise subscription](https://octopus.com/enterprise).
```

Comment:
Can this be updated to point to the pricing page? The enterprise page doesn't talk about the enterprise tier at all yet as it's a soft launch.

Reply:
i.e. `https://octopus.com/pricing`

Reply:
```suggestion
The ServiceNow Integration feature is available from Octopus **2022.3** onwards and requires an [enterprise subscription](https://octopus.com/pricing).
```

Reply:
Done, thanks @robpearson